pkg/database/migrations/021-update_query_order.sql

```diff
@@ -0,0 +1,28 @@
+-- This file is Free Software under the Apache-2.0 License
+-- without warranty, see README.md and LICENSES/Apache-2.0.txt for details.
+--
+-- SPDX-License-Identifier: Apache-2.0
+--
+-- SPDX-FileCopyrightText: 2024 German Federal Office for Information Security (BSI) <https://www.bsi.bund.de>
+-- Software-Engineering: 2024 Intevation GmbH <https://intevation.de>
+
+
+DO $$
+DECLARE
+    default_definer constant varchar = 'system-default';
+    default_advisory_columns text array default Array['cvss_v3_score', 'cvss_v2_score', 'comments', 'critical', 'id', 'recent', 'versions', 'title', 'publisher', 'ssvc', 'state', 'tracking_id'];
+    default_event_columns text array default Array['cvss_v3_score', 'cvss_v2_score', 'comments', 'critical', 'id', 'title', 'publisher', 'ssvc', 'tracking_id', 'event', 'event_state', 'time', 'actor', 'comments_id'];
+BEGIN
+    DELETE FROM stored_queries WHERE definer = default_definer;
```

Comment:
You should really only delete what you want to replace. It could be possible that someone else has inserted data with the `system-default` user which would be destroyed by this, too.